packages/translations/src/i18next/i18next.ts

```diff
@@ -1,6 +1,6 @@
 import { str as crc32 } from 'crc-32';
 import i18n from 'i18next';
-import { initReactI18next } from 'react-i18next';
+import { useTranslation, initReactI18next } from 'react-i18next';
```

Comment:
you have imported `useTranslation` but not used it in the file.
maybe you wanted to destructure `i18n` from `useTranslation` instead of importing it from `i18next` ? :thinking:

Reply:
Oh man, I was trying different ways to solve one issue we had in this earlier. will remove it, no more needed. Thanks man 🙏